package/lib/src/beam_guard.dart

```diff
@@ -79,6 +79,8 @@ class BeamGuard {
   ///
   /// If asterisk is present, it is enough that the pre-asterisk substring is
   /// contained within location's pathBlueprint.
+  /// If the uri has a query, it is enough that the pre-query substring is
+  /// equal to the pathBlueprint.
```

Comment:
Maybe we should now say "path" instead of "pre-query", or mention both terms for clarity, "...pre-query substring, i.e. path..."